certified-connectors/Plumsail Forms/apiDefinition.swagger.json

```diff
@@ -1,14 +1,14 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
     "title": "Plumsail Forms",
     "description": "This API allows you to process Plumsail Forms submissions and their attachments.",
```

Comment:
@jusper-dk please remove the word "API" from the description. please follow these [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)

Reply:
Fixed.